Formula/pyqt.rb

```diff
@@ -24,19 +24,20 @@ class Pyqt < Formula
 
   def install
     python = Formula["python@3.9"]
+    qt = Formula["qt"]
     args = %W[
       --target-dir #{prefix}
       --no-make
       --confirm-license
     ]
     system "sip-build", *args
     cd "build" do
-      qt_prefix = Formula["qt"].prefix Formula["qt"].version
+      qt_prefix = HOMEBREW_CELLAR/"qt"/qt.version
```

Comment:
Actually, you know what, I think you were right the first time. Let's just make it clearer you're using `qt.version` as an argument to `qt.prefix` here:
```suggestion
      qt_prefix = qt.prefix(qt.version)
```
Sorry for flip-flopping here!

Reply:
Squashed.

Reply:
Thanks for your patience!